### PR TITLE
[stari_list_3]: list学习day2.

### PR DESCRIPTION
**Pull request by XavierZXY**
### What?

- brief: 更新成员关于list学习的内容
- author:zxy_xavier@163.com

### What changes are included in this PR?
- zxy
  - 添加`task_list_1.md`，记录部分学习记录
- bruca_jay
  - 添加`STL_note.md`，记录部分学习记录
- gyffffff
  - 记录部分学习记录

### Are these changes tested?
部分测试

### CODE REVIEW

File: stl/stari/gyfffffff/src/task_1/test1_1.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// to have a look of  the size of any list
+// can be revise to a template function ...
+#include <iostream>
+#include <list>
+
+using std::cout;
```
Comment (Light-City) [inline stl/stari/gyfffffff/src/task_1/test1_1.cpp:6]: global namespace

Comment (Light-City) [reply stl/stari/gyfffffff/src/task_1/test1_1.cpp:6]: ban!!!

File: stl/stari/gyfffffff/src/task_1/test1_2.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// test a list is a recursive or not
+// the program still exits some problem ... for it doesn't do as excepted
+
+#include <iostream>
+#include <list>
+
+using std::cout;
+using std::endl;
```
Comment (Light-City) [inline stl/stari/gyfffffff/src/task_1/test1_2.cpp:8]: global namespace.

ban!!!!

File: stl/stari/bruce_jay/images/image-20230619223159513.png
Comment (Light-City) [inline stl/stari/bruce_jay/images/image-20230619223159513.png:1]: img name Variable names should make sense

File: stl/stari/gyfffffff/docs/images/1_1.png
Comment (Light-City) [inline stl/stari/gyfffffff/docs/images/1_1.png:1]: 1_1.png or 1_2.png should make sense.

Comment (XavierZXY) [reply stl/stari/gyfffffff/docs/images/1_1.png:1]: 更新学习内容

Comment (Light-City) [reply stl/stari/gyfffffff/docs/images/1_1.png:1]: did not see the change

**Comment by Light-City** [comment]
stari_list_3 -> stari_list_1

**Comment by Light-City** [comment]
Time has expired, eliminated, refused to merge

**Pull request closed without merging**